src/radiation_system.hpp

```diff
@@ -1199,7 +1196,7 @@ void RadSystem<problem_t>::AddSourceTerms(array_t &consVar, arrayconst_t &radEne
 					lorentz_factor_v = lorentz_factor;
 					lorentz_factor_v_v = lorentz_factor;
 				} else {
-					AMREX_ASSERT(false);
+					AMREX_ALWAYS_ASSERT_WITH_MESSAGE(false, "Invalid beta_order_ value.");
```

Comment:
Can you change this to `static_assert(beta_order == 0 || beta_order == 1)`?

Reply:
OK. Done.